### PR TITLE
[M0-2Vlan] Enable copp test on M0-2Vlan topo

### PR DESCRIPTION
**Pull request by lizhijianrd**
### Description of PR

Summary:
Enable copp test on m0-2vlan topo.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311
- [x] 202405

#### What is the motivation for this PR?
Enable copp test on m0-2vlan topo.

#### How did you do it?
Update conditional mark file and copp test pytest mark.

#### How did you verify/test it?
Verified on Nokia-7215 M0-2vlan testbed.

### CODE REVIEW

**Review by Gfrom2016 — APPROVED** [review]
LGTM

File: tests/copp/test_copp.py [hunk 1]
```diff
@@ -43,7 +43,7 @@
 from tests.common.fixtures.ptfhost_utils import change_mac_addresses      # noqa F401
 
 pytestmark = [
-    pytest.mark.topology("t0", "t1", "t2", "m0", "mx"),
+    pytest.mark.topology("t0", "t1", "t2", "m0", "m0-2vlan", "mx"),
```
Comment (yaqiangz) [inline tests/copp/test_copp.py:46]: Is this change needed? I think m0 is ok?

Comment (lizhijianrd) [reply tests/copp/test_copp.py:46]: I tried without this change. All testcases are skipped. So I think it should be necessary.

Comment (lizhijianrd) [reply tests/copp/test_copp.py:46]: I see, we can set `--topoloty` to m0 when run test, then no need this change.

Comment (lizhijianrd) [reply tests/copp/test_copp.py]: Updated the code.

**Review by yaqiangz — APPROVED** [review]

**Review by yutongzhang-microsoft — APPROVED** [review]
LGTM

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Comment by lizhijianrd** [comment]
> @lizhijianrd PR conflicts with 202311 branch

Manually created backport PR: https://github.com/sonic-net/sonic-mgmt/pull/13725

**Comment by lizhijianrd** [comment]
Backport 202311 PR #13725 has been merged.
